focusMode.js

```diff
@@ -0,0 +1,21 @@
+// Variable to track focus mode state
+let isFocusModeOn = false;
+
+function setFocusMode(data) {
+  const focusMode = data.focusMode;
+  console.log(focusMode);
+  isFocusModeOn = !isFocusModeOn;
+
+  if (isFocusModeOn) {
+    focusModeButton.textContent = "Deactivate Focus Mode";
+    console.log("Focus Mode is now ON");
+  } else {
+    focusModeButton.textContent = "Activate Focus Mode";
+    console.log("Focus Mode is now OFF");
+  }
+}
+
+// JavaScript
+const focusModeButton = document.getElementById("focusModeButton");
```

Comment:
This code is not needed as you have already specified the setFocusMode function in script.js. Also, it won't work as it is looking for the button in the main page, not in the popup